credentials.json

```diff
@@ -0,0 +1 @@
+{"web":{"client_id":"594743792021-tvffn11n961cqea8mbufm9gnts372m0n.apps.googleusercontent.com","project_id":"anthrolink-1554453632480","auth_uri":"https://accounts.google.com/o/oauth2/auth","token_uri":"https://oauth2.googleapis.com/token","auth_provider_x509_cert_url":"https://www.googleapis.com/oauth2/v1/certs","client_secret":"8aant_BOs6holQms7d8PM_wN","javascript_origins":["https://localhost:8080"]}}
```

Comment:
Please remove credentials.json it is a private file. Moreover this can change when  we have to give different user's Google Api access